behave/helper/helper_base.py

```diff
@@ -29,6 +30,12 @@ def maximize(self):
     def close(self):
         self._driver.quit()
 
+    def get_date_now(self):
```

Comment:
Update this if you update the name in __init__

Reply:
Changed.